README.md

```diff
@@ -1 +1,69 @@
 # Professional README Generator
+
+![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)
+<br>
+
+## Description
+
+A command-line application which accepts user's input and based on it dynamically generates a professional README file for a new project using the inquirer package. The newly generated README will be found in the assets/js folder.
+<br>
+<br>
+
+## Motivation
+
+The purpose of this app is to practice working with node.js, npm, inquirer, fs, and more.
+<br>
+<br>
+Link to the deployed app: [here](https://jitasek.github.io/generate-readme/)
+<br>
+Link to the video guide: [here](https://drive.google.com/file/d/1E0dAvRmjQUrE5KG02HClb8i7FoRieMIm/view)
+
+(If the video quaility on the link above is not sufficient for you, please download the video guide file from the repo and play it locally.)
+<br>
+<br>
+
+## User Story
+
+As a developer I want a README generator so that I can quickly create a professional README for a new project.
+<br>
+<br>
+
+## Installation and Usage
+
+Clone this repository to your local environment using the following command in your terminal:
+
+`git clone git@github.com:jitasek/generate-readme.git`
+
+Install all the dependencies defined in package.json to your local folder using the following command:
+
+`npm install`
+
+Position yourself in the js folder (cd assets/js) where the main index.js file is located and run in the command line:
+
+`node index.js`
+
+You will get prompted 9 times for input. After you have selected your option(s) in the last prompt, press enter and a new README file will get generated with your answers in it. You will then find it in the js folder.
+<br>
+<br>
+
+## Acceptance Criteria
```

Comment:
this could do nicely under the user story heading

Reply:
Makes sense, thanks!